src/resources/js/components/Presentation.vue

```diff
@@ -45,17 +45,16 @@ export default {
       polls: [],
     };
   },
+  watch: {
```

Comment:
found this solution on https://router.vuejs.org/guide/essentials/dynamic-matching.html#reacting-to-params-changes